src/ethereum_clis/types.py

```diff
@@ -93,3 +101,19 @@ class TransitionToolOutput(CamelModel):
     alloc: Alloc
     result: Result
     body: Bytes | None = None
+
+
+class TransitionToolState(CamelModel):
+    """Transition tool state."""
+
+    fork: str
+    chain_id: int = Field(..., alias="chainid")
+    reward: int
+    blob_schedule: BlobSchedule | None
+
+
+class TransitionToolPost(CamelModel):
```

Comment:
it has input, but again reading Post I would expect it to be post Alloc

Reply:
Ah, this was named after the post request (as in html) that is made to the t8n server when requesting a state transition.
Bad name decision, changing it to `TransitionToolRequest` 👍